### PR TITLE
TTRSS update blog post.

### PR DESCRIPTION
**Pull request by zenhack**
This is still WIP; among other things it ends way too suddenly; need to put in a conclusion or something. Marking it as a draft PR, but feedback welcome. (I'll obviously need to bump the date to whenever we actually merge this).

### CODE REVIEW

File: _posts/2020-08-04-tiny-tiny-rss-and-the-powerbox.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  year) to check for updates periodically even when the user isn't
+  actively using the app. This was a major pain point for some users,
+  as it meant that if they didn't check their feeds often enough, they
+  could miss articles.
+- I've migrated the package away from using Sandstorm's deprecated
+  HackSession interface for fetching feeds, and over to using the
+  Powerbox.
+
+The latter of these is important to enable Sandstorm's goal of full
+network-isolation for applications. It has always been the intent that
```
Comment (ocdtrekkie) [inline _posts/2020-08-04-tiny-tiny-rss-and-the-powerbox.md:22]: This dash seems extraneous.

File: _posts/2020-08-04-tiny-tiny-rss-and-the-powerbox.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+access to the relevant hosts on-demand. The daemon is re-usable and
+could be used as a quick way to port other existing apps that have
+legitimate need to access a variety of different hosts at runtime,
+which are not known in advance.
+
+The daemon is exciting in that it can make basic porting of apps not
+designed for Sandstorm much quicker, but the integration is imperfect: as
+it stands, the user will see an extra prompt (or two) when first
+subscribing to a feed. Maintaining security while avoiding annoying extra
+prompts like this [is one of the design goals][2] of sandstorm's powerbox,
```
Comment (ocdtrekkie) [inline _posts/2020-08-04-tiny-tiny-rss-and-the-powerbox.md:42]: s/sandstorm/Sandstorm here

**Comment by zenhack** [comment]
Knew I was missing something. Probably we should also squeeze in some discussion of further enhancements to the powerbox; right now the way the end is worded makes it sound like all of the problems can be solved inside the app, which doesn't acknowledge things like sandstorm-io/sandstorm#3396, or indeed that the fact that right now we have no way of poking the user about powerbox requests when they don't have a browser window open (maybe we could rig up notifications or such?)

**Comment by zenhack** [comment]
Added bits about the limitations, and fixed the clerical issues; PTAL.

**Comment by zenhack** [comment]
(I also bumped the date to tomorrow, though we can push it further back if it takes longer than that to land).

**Comment by zenhack** [comment]
Cool. @kentonv, when you've got the time...

**Comment by kentonv** [comment]
Published.

**Pull request merged**